apps/web-giddh/src/app/subscription/subscription-list/subscription-list.component.ts

```diff
@@ -520,10 +509,12 @@ export class SubscriptionListComponent implements OnInit, OnDestroy {
         this.showCountry = false;
         this.showPlanSubName = false;
         this.showMonthlyYearly = false;
-        this.showStatus = false;
         this.subscriptionListForm.reset();
         this.inlineSearch = '';
-        this.getAllSubscriptions(true);
+        this.selectedStatus = 'Active';
```

Comment:
```suggestion
        this.selectedStatus = this.localeData?.status_active;
```

also this.statusOptions?.[0]?.label